src/arx7.js

```diff
@@ -119,6 +121,26 @@ export class ARX7 {
           this.client.say(from, `I'm a bot! Contact [${admins}] for help`);
           resolve();
         }
+      }
+
+      const helpRegex = /^[.!](?:h(?:elp)?|man)(?:\s(.+))?$/i;
+      const help = text.match(helpRegex);
+
+      if (help) {
+        if (help[1] === undefined) {
+          log(`Help request from ${from}`);
+          const str = `Syntax: .help [module]. Modules: ${this.moduleList()}.`;
+          this.client.say(to, str);
```

Comment:
Usually messages meant for a single user that aren't useful to the wider public are just better sent as notices.